src/quacc/recipes/newtonnet/ts.py

```diff
@@ -293,3 +307,250 @@ def _get_hessian(atoms: Atoms) -> NDArray:
     ml_calculator.calculate(atoms)
 
     return ml_calculator.results["hessian"].reshape((-1, 3 * len(atoms)))
+
+
+"""
+@job
+@requires(NewtonNet, "NewtonNet must be installed. Refer to the quacc documentation.")
+def neb_job(
+    atoms: Atoms,
+    relax_endpoints: bool = True,
+    run_geodesic: bool = True,
+    run_single_ended: bool = True,
+    run_freq: bool = True,
+    neb_job_kwargs: dict[str, Any] | None = None,
+    relax_job_kwargs: dict[str, Any] | None = None,
+    freq_job_kwargs: dict[str, Any] | None = None,
+):
+    relax_job_kwargs = relax_job_kwargs or {}
+    freq_job_kwargs = freq_job_kwargs or {}
+
+    irc_job_defaults = {"max_steps": 5}
+    irc_job_kwargs = recursive_dict_merge(irc_job_defaults, irc_job_kwargs)
+
+    # Run IRC
+    irc_summary = strip_decorator(irc_job)(
+        atoms, direction=direction, run_freq=False, **irc_job_kwargs
+    )
+
+    # Run opt
+    relax_summary = strip_decorator(relax_job)(irc_summary["atoms"], **relax_job_kwargs)
+
+    # Run frequency
+    freq_summary = (
+        strip_decorator(freq_job)(relax_summary["atoms"], **freq_job_kwargs)
+        if run_freq
+        else None
+    )
+    relax_summary["freq_job"] = freq_summary
+    relax_summary["irc_job"] = irc_summary
+
+    return relax_summary
+"""
+
+
+def sella_wrapper(
```

Comment:
Haha! Yes, it should not have existed in the first place. I had it because I was running things independently and I moved the entire infrastructure to this branch in `QuAcc`. I will trying to replace things one by one making sure that the tests run after each modification.